### PR TITLE
Fix profile-rt licensing

### PR DESCRIPTION
**Pull request by JohanEngelen**
- Relicense profile-rt/d/ldc/profile.d (sole author Johan Engelen).
- Add profile-rt license to LICENSE.

See https://forum.dlang.org/thread/kmkusabahjitopkhtgae@forum.dlang.org

### CODE REVIEW

**Comment by JohanEngelen** [comment]
@redstar ping

**Comment by JohanEngelen** [comment]
Maybe, I am a compiler-rt contributor though.

**Comment by JohanEngelen** [comment]
Will merge soon if no further comments.

File: runtime/profile-rt/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+===============================
+
+`profile-rt` is the runtime library for writing profiling instrumentation
+files. It is linked with instrumented executables (`-fprofile-instr-generate`).
+
+`profile-rt` consists of two parts: a part from LLVM (C/C++,
+`profile-rt/profile-rt-3*`), and a D interface to the lib (`profile-rt/d`).
+See the source files and LDC's LICENSE file for licensing details.
+
+The sources in `profile-rt/profile-rt-3*` are exact copies of the `lib/profile
```
Comment (kinke) [inline runtime/profile-rt/README.md]: Missing trailing `.

**Pull request merged**